powerlevel9k.zsh-theme

```diff
@@ -429,10 +429,12 @@ prompt_dir() {
   fi
 
   local current_icon=''
-  if [[ $(print -P "%~") == '~'* ]]; then
+  if [[ $(print -P "%~") == '~' ]]; then
     "$1_prompt_segment" "$0_HOME" "$2" "blue" "$DEFAULT_COLOR" "$current_path" 'HOME_ICON'
-  else
+  elif [[ $(print -P "%~") == '~'* ]]; then
     "$1_prompt_segment" "$0_DEFAULT" "$2" "blue" "$DEFAULT_COLOR" "$current_path" 'FOLDER_ICON'
+  else
+    "$1_prompt_segment" "$0_HOME" "$2" "blue" "$DEFAULT_COLOR" "$current_path" 'HOME_SUB_ICON'
```

Comment:
This is the third state. So it should be `FOLDER_ICON` and instead of `$_HOME` it should be `$0_DEFAULT`.